cli/internal/taskhash/taskhash_test.go

```diff
@@ -113,14 +109,15 @@ func Test_manuallyHashPackage(t *testing.T) {
 	}
 
 	count = 0
-	justFileHashes, err := manuallyHashPackage(pkg, []string{filepath.FromSlash("**/*file")}, turbopath.AbsoluteSystemPath(repoRoot.ToString()))
+	justFileHashes, err := manuallyHashPackage(pkg, []string{filepath.FromSlash("**/*file")}, repoRoot)
 	if err != nil {
 		t.Fatalf("failed to calculate manual hashes: %v", err)
 	}
 	for path, spec := range files {
-		if strings.HasPrefix(path.ToString(), prefix.ToString()) {
+		if path.ToSystemPath().HasPrefix(pkgName) {
```

Comment:
This was previously comparing an `AnchoredUnixPath` and an `AnchoredSystemPath`. 🤦‍♂️

So now we have a utility function to stop doing that.